README.md

```diff
@@ -277,22 +264,24 @@ to unset the `editor.defaultFormatter` in `settings.json`:
 }
 ```
 
-### Enabling the Rust-based language server
+### Using the Rust-based language server
 
-To use the new Rust-based language server (`ruff server`), you'll need to enable the "Native Server"
-setting in the extension settings:
+The Ruff extension will automatically use the Rust-based language server (`ruff server`) if the `ruff`
+executable is at least version `0.5.3` and the `nativeServer` setting is set to `auto` (default). You can explicitly
```

Comment:
I think the actual implementation also considers whether the user uses any ruff-lsp only settings. Should we mention this here as well?